Description: Make soname be the major version
 ndn ships with soname == version, that is going to make a lot of packages,
 change that so that soname matches only the major version, we'll go bumping
 as needed
 .
 libndn-cxx (0.5.0+git20161122-2) unstable; urgency=medium
 .
   * New Upstream release
Author: Niv Sardi <xaiki@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2016-12-10

--- libndn-cxx-0.5.0+git20161122.orig/wscript
+++ libndn-cxx-0.5.0+git20161122/wscript
@@ -184,7 +184,7 @@ def build(bld):
     if bld.env.enable_shared:
         bld(features=['pch', 'cxx', 'cxxshlib'],
             vnum=VERSION_BASE,
-            cnum=VERSION_BASE,
+            cnum=VERSION_BASE.split('.')[0],
             **libndn_cxx)
 
     if bld.env.enable_static:
